### DISCUSSION TITLE
Repo structure

### DISCUSSION THREAD

**Discussion by zy2ba** [TinkoffCreditSystems/shiva#32] [Ideas]
At now apps have not best names.
At first look I recignized demo as example of usage, but this is just docs portal. At other hand "host" and "remote" nothing to say for newcomers. 
May be shoud add directory "examples" and migrate "root" and "host" to it? Also optonally may rename demo to docs

**Comment by IKatsuba** [comment]
It's a good idea! Thanks, @zy2ba!
